docs/root/intro/arch_overview/observability/statistics.rst

```diff
@@ -37,4 +37,10 @@ Internally, counters and gauges are batched and periodically flushed to improve
 Histograms are written as they are received. Note: what were previously referred to as timers have
 become histograms as the only difference between the two representations was the units.
 
+Lazy Init of Stats
+=================
+
+When :ref:`enable_lazyinit_stats <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.enable_lazyinit_stats>` is enabled in Bootstrap, for stats that supports the LazyCompatibleInterface, 
+the actual stats struct creation is deferred to first access of any member of that stats. E.g. when enabled, A ClusterTrafficStats structure is only created when any member of ClusterInfoImpl::trafficStats() is accessed.
```

Comment:
I think we should enumerate them here. Otherwise a user will have to grep through the code to know which ones this impacts.